apps/entity-service/src/entities/dto/association.dto.ts

```diff
@@ -1,9 +1,9 @@
 import { JsonApiAttributes } from "@terramatch-microservices/common/dto/json-api-attributes";
 
-export type AssociationDtoAdditionalProps = {
+export interface AssociationDtoAdditionalProps {
```

Comment:
Doesn't matter much, but I'm curious why you moved this to an interface?

Reply:
I wanted to extend this one for Media